code/modules/reagents/reagents/medicine.dm

```diff
@@ -128,7 +128,7 @@
 	M.adjustToxLoss(-((0.4 + (M.getToxLoss() * 0.05)) * effect_multiplier))
 	M.add_chemical_effect(CE_ANTITOX, 1)
 	holder.remove_reagent("pararein", 0.8 * effect_multiplier)
-	holder.remove_reagent("blattedin", 0.4 * effect_multiplier)
+	holder.remove_reagent("carpotoxin", 0.4 * effect_multiplier) // Gonna be good for fish recipes
 	holder.remove_reagent("toxin", 0.4 * effect_multiplier)
```

Comment:
```suggestion
	holder.remove_reagent("toxin", 0.4 * effect_multiplier)
	holder.remove_reagent("blattedin", 0.4 * effect_multiplier)
```

Reply:
Please refer to Blattedin code and how its `affect_blood` overrides its toxin parent, it's completely innocuous for humans, only affects roaches for revival and Cht'mant for healing, and this helps applying Dylovene to Cht'mant as well as Blattedin for better cohesive healing without one stepping over the other. It makes more sense for Dylovene to remove an actual toxic component that just mere roach farts on the blood, and this change helps Cht'mant as well, who are well a pain in the tail to fix already.